### PR TITLE
Drawer navigation

### PR DESCRIPTION
**Pull request by Tyratox**
This is a small PR replacing the bottom tab navigation with a drawer navigation.

Screenshot 2022-11-24 at 08 44 01

### CODE REVIEW

File: fe1-web/src/features/digital-cash/components/DigitalCashWalletItems.tsx [hunk 1]
```diff
@@ -1,39 +1,24 @@
...
   // the digital cash wallet item should be at the top
   const listStyles = List.getListItemStyles(true, false);
 
   return (
     <>
-      <ListItem
-        containerStyle={listStyles}
-        style={listStyles}
-        onPress={() => navigation.navigate(STRINGS.navigation_wallet_digital_cash_wallet)}>
+      <ListItem containerStyle={listStyles} style={listStyles} onPress={() => {}}>
```
Comment (pierluca) [inline fe1-web/src/features/digital-cash/components/DigitalCashWalletItems.tsx:21]: onPress -> do nothing ?

**Review by pierluca — COMMENTED** [review]
Minor comment, looks good otherwise 👍

File: fe1-web/src/features/digital-cash/components/DigitalCashWalletItems.tsx (hunk 1 above)
Comment (Tyratox) [reply fe1-web/src/features/digital-cash/components/DigitalCashWalletItems.tsx:21]: Whoops, good catch! This component should have been deleted!

**Review by pierluca — DISMISSED** [review]

**Review by pierluca — APPROVED** [review]
Later on we might actually want to tag the repo and display the version number, but this is good enough for now 👍

File: fe1-web/src/core/components/PoPIcon.tsx [hunk 2]
```diff
@@ -30,6 +30,13 @@ type MaterialIcon = {
   IconFamily: typeof MaterialIcons;
 };
 
+type AndDesignIconNames = keyof typeof AntDesign['glyphMap'];
```
[the remark this answers is not in the document]
Comment (Tyratox) [reply fe1-web/src/core/components/PoPIcon.tsx]: I guess my subconsciousness realized 🐜 are bugs and this software should be free of that? 😂

**Comment by Tyratox** [comment]
@pierluca What do you think regarding the security hotspots? It would be a problem if the attacker could control the PATH variable or if somebody did not have `git` in their PATH and an attacker would place an executable called `git` inside the working directory on a developer's machine. Personally I would asses this as low (additional) risk given we are using git for version control and are executing git commands all the time. But the impact would of course still be big.

File: fe1-web/src/core/components/PoPIcon.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (pierluca) [reply fe1-web/src/core/components/PoPIcon.tsx]: 🤣

**Comment by pierluca** [comment]
Agreed. Safe enough 😄

**Review by pierluca — COMMENTED** [review]
I'm not sure what you did, but somehow when you merged master in your branch, you brought in a lot of changes that appear as new files 🤔  Are you sure you want to do that ?

**Comment by Tyratox** [comment]
Might very well be because of the fever but at the moment I can't see what you mean 😅 When clicking on "Files changed" it seems not too many changes appear as new files (at least not more than I would expect)

**Review by pierluca — APPROVED** [review]

**Pull request merged**